packages/main/src/Link.ts

```diff
@@ -220,6 +225,36 @@ class Link extends UI5Element implements ITabbable {
 	@property({ type: Object })
 	accessibilityAttributes!: LinkAccessibilityAttributes;
 
+	/**
+	 * Defines the icon, displayed as graphical element within the component before the link's text.
+	 * The SAP-icons font provides numerous options.
+	 *
+	 * **Note:** We recommend using аn icon in the beginning or the end only. Using two icons is discouraged.
+	 *
+	 * Example:
```

Comment:
What is the purpose of "Example:" here?

Same for the other property

Reply:
Missed that, will remove it. Thanks